bigtable/beam/helloworld/pom.xml

```diff
@@ -37,7 +37,10 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <apache_beam.version>2.31.0</apache_beam.version>
+    <!-- Keep this apache_beam.version in sync with version used in bigtable-hbase-beam -->
+    <!-- https://mvnrepository.com/artifact/com.google.cloud.bigtable/bigtable-hbase-beam -->
+    <apache_beam.version>2.24.0</apache_beam.version>
+    <exec.cleanupDaemonThreads>false</exec.cleanupDaemonThreads>
```

Comment:
what's this for?

Reply:
When you run it locally it prevents a bunch of warnings about threads